db/people.yml

```diff
@@ -1055,10 +1055,25 @@ cyshi:
   position: scientist
 dbrady:
   active: true
-  aka: []
+  aka:
+    - David Kim
+    - DK
+    - David (Spanish pronunciation)
   avatar: "https://github.com/davidkim1.png"
-  bio: bio
-  education: []
+  bio:
```

Comment:
I think bio should a string not a list.

There is another field called `bios` where you can store different bios that can then be accessed later, but by default the web-page builds with this bio which should be a string.